code/game/objects/items/implants/implantchair.dm

```diff
@@ -171,7 +171,7 @@
 		return FALSE
 	H.set_species(/datum/species/human, 1)//lizards go home
 	purrbation_remove(H)//remove cats
-	H.dna.remove_all_mutations()//hulks out
+	H.dna.remove_all_mutations()//powergamers out
```

Comment:
Just remove the comment, neither this nor the old one helps make the code any more readable. Comments should explain why, not what.

Reply:
Removed it.